src/json/lexer.cr

```diff
@@ -154,7 +154,7 @@ abstract class JSON::Lexer
       when '\0'
         raise "Unterminated string"
       when '\\'
-        @buffer << consume_string_escape_sequence
+        consume_string_escape_sequence
```

Comment:
Why this isn't using skip: true?

Reply:
Because it's inside `consume_string_with_buffer`, which cares about collecting the contents of the string into a buffer. Only `consume_string_skip` uses `skip: true`.